framework/branch-key-store.md

```diff
@@ -582,73 +744,71 @@ and the `branchKeyId` from the returned `branch-key-id` field.
 
 This operation MUST return the constructed [beacon key materials](./structures.md#beacon-key-materials).
 
-## Encryption Context
+## Branch Key Context
 
-This section describes how the AWS KMS encryption context is built
-from the DynamoDB items that store the branch keys.
+Branch Key Context is a set of key value pairs (a Map) that contain contextual information about the Branch Key Item.
 
-The following encryption context keys are shared:
+The Branch Key Context:
 
-- MUST have a `branch-key-id` attribute
-- The `branch-key-id` field MUST not be an empty string
-- MUST have a `type` attribute
-- The `type` field MUST not be an empty string
-- MUST have a `create-time` attribute
-- MUST have a `tablename` attribute to store the logicalKeyStoreName
-- MUST have a `kms-arn` attribute
-- MUST have a `hierarchy-version`
-- MUST NOT have a `enc` attribute
+- MUST have a `branch-key-id` key who's value MUST not be an empty string
+- MUST have a `type` key who's value MUST not be an empty string
+- MUST have a `create-time` key with a value in ISO 8601 format in UTC
+- MUST have a `tablename` key who's value is the logicalKeyStoreName
+- MUST have a `kms-arn` key who's value is valid KMS ARN
+- MUST have a `hierarchy-version` key who's value is either "1" or "2"
+- MUST NOT have a `enc` key
+- MAY have one or more keys prefixed with `aws-crypto-ec:` which is the encyption context send by the customer.
+  The `aws-crypto-ec:` prefix is prepended by the library
+- MUST NOT have any other keys apart from the ones mentioned above if `hierarchy-version` is "2"
 
-Any additionally attributes on the DynamoDB item
-MUST be added to the encryption context.
-
-### ACTIVE Encryption Context
+### ACTIVE Branch Key Context
 
 The ACTIVE branch key is a copy of the DECRYPT_ONLY with the same `version`.
 It is structured slightly differently so that the active version can be accessed quickly.
```

Comment:
```suggestion
The ACTIVE branch key is a copy of a DECRYPT_ONLY with the same `version`.
It is structured slightly differently so that the active version can be accessed quickly.
```